Library/Homebrew/os/linux/glibc.rb

```diff
@@ -0,0 +1,14 @@
+module OS
+  module Linux
+    module Glibc
```

Comment:
Could this just be required from `system_config` or even just be in that file itself if it's only used there?

Reply:
Yes, it can be required by `system_config`. Done.
This method will be used later by the bottle-pouring logic to ensure that a new enough `glibc` is installed to use bottles.

Reply:
Cool, that's fine.